src/service/randomNumberInRange.js

```diff
@@ -0,0 +1,41 @@
+// min should be a minimal number, max should be a maximal number, forbiddenNumbersArray should be an array containing all the numbers that should not be generated
+
+export const randomNumberInRange = (min, max, forbiddenNumbersArray = []) => {
+    
+    if (typeof min !== 'number' || typeof max !== 'number' || !Array.isArray(forbiddenNumbersArray) || !forbiddenNumbersArray.every(number => typeof number === 'number')) {
```

Comment:
Dobrze byłoby zastosowac np. Prettier i usawić max długośc linii - strasznie ciężko się to czyta, jak trzeba przewijać.